doc/tutorials/basic/build_animation.md

```diff
@@ -2,11 +2,22 @@
 
 In this tutorial, we will create a *bar chart race* using the [Altair](https://altair-viz.github.io) plotting library and the [`Player`](../../reference/widgets/Player.ipynb) widget.
 
+*Todo: The below is for testing on ios. Clean up*
```

Comment:
This was updated temporarily to demonstrate that mp4 videos hosted on aws don't play on iphone while hosted as part of repository they do. Its the same issue for all mp4 videos used in Panel docs.

Someone needs to understand and fix. My hypothesis from google search is that its a *certificate* issue on aws.

Reply:
Doubt it's a certificate issue, my best guess would be the Content-type headers not being set correctly.